python/delta/tests/test_sql.py

```diff
@@ -104,6 +104,7 @@ def read_table() -> DataFrame:
                 return self.spark.sql(f"SELECT * FROM {table}")
 
             self.spark.sql(f"DROP TABLE IF EXISTS {table}")
```

Comment:
I think this should be added anyway since we create `table` and `table_part` down below